quick_start/kusto_sample_app.py

```diff
@@ -0,0 +1,334 @@
+    '{"Properties":{"Path":"$.xint64"},"column":"xint64","datatype":"long"},'
+    '{"Properties":{"Path":"$.xuint8"},"column":"xuint8","datatype":"long"},'
+    '{"Properties":{"Path":"$.xuint16"},"column":"xuint16","datatype":"long"},'
+    '{"Properties":{"Path":"$.xuint32"},"column":"xuint32","datatype":"long"},'
+    '{"Properties":{"Path":"$.xuint64"},"column":"xuint64","datatype":"long"},'
+    '{"Properties":{"Path":"$.xdate"},"column":"xdate","datatype":"datetime"},'
+    '{"Properties":{"Path":"$.xsmalltext"},"column":"xsmalltext","datatype":"string"},'
+    '{"Properties":{"Path":"$.xtext"},"column":"xtext","datatype":"string"},'
+    '{"Properties":{"Path":"$.rowguid"},"column":"xnumberAsText","datatype":"string"},'
+    '{"Properties":{"Path":"$.xtime"},"column":"xtime","datatype":"timespan"},'
+    '{"Properties":{"Path":"$.xtextWithNulls"},"column":"xtextWithNulls","datatype":"string"},'
+    '{"Properties":{"Path":"$.xdynamicWithNulls"},"column":"xdynamicWithNulls","datatype":"dynamic"}]'
+)
+csvSample = "dataset.csv"
+jsonSample = "dataset.json"
+
+# Todo - Config (Optional): Change the authentication method from Device Code (User Prompt) to one of the other options
+#  Some of the auth modes require additional environment variables to be set in order to work (check the use below)
+#  Managed Identity Authentication only works when running as an Azure service (webapp, function, etc.)
+authenticationMode = "userPrompt"  # choose between: (userPrompt|managedIdentity|AppKey|AppCertificate)
+waitForUser = True
+
+
+def main():
+    print("Kusto sample app is starting...")
+    if authenticationMode == "userPrompt":
+        print("")
+        print("You will be prompted for credentials during this script.")
+        print("Please, return to the console after authenticating.")
+        wait_for_user()
+
+    # Todo - Tip: Avoid creating a new Kusto Client for each use.
+    #  Create the clients once and use them as long as possible.
+    kusto_connection_string = create_connection_string(kustoUri, authenticationMode)
+    ingest_connection_string = create_connection_string(ingestUri, authenticationMode)
+    kusto_client = KustoClient(kusto_connection_string)
+    ingest_client = QueuedIngestClient(ingest_connection_string)
+
+    print("")
+    print(f"Creating table '{databaseName}.{tableName}' if it does not exist:")
+    # Todo - Tip: this is commonly a one-time command
+    # Todo - Learn More: For additional information on how to create tables see: https://docs.microsoft.com/en-us/azure/data-explorer/one-click-table
+    command = f".create table {tableName} {tableSchema}"
+    if not run_control_command(kusto_client, databaseName, command):
+        print("Failed to create or validate table exists.")
+        exit(-1)
+
+    wait_for_user()
+
+    print("")
+    print(f"Altering the batching policy for '{tableName}'")
+    # Todo - Tip: this is commonly a one-time command

```

Comment:
Specify this is for testing only

Reply:
There's a whole section explaining it in line 88 on the latest version.
LMK if you think it needs more info